content/news/052/index.md

```diff
@@ -117,7 +143,7 @@ and highlight events from the past. -->
 
 <!-- Section to get more people involved in writing news. -->
 
-Editing this newsletter wouldn't be possible without [your contributions][news_current_prs]. 
```

Comment:
Argh, meaningful trailing whitespace is my least favourite part of the markdown spec - always messes with my "auto remove trailing whitespace by default" setting in my editor.

May I add an [`.editorconfig`](https://editorconfig.org/) to this repo so as to discourage editors from removing trailing whitespace in markdown files?

Reply:
Good idea! I added one [`c385b61` (#1529)](https://github.com/rust-gamedev/rust-gamedev.github.io/pull/1529/commits/c385b618911e0e298043bca048348a267d2afdd4)